vulnfeeds/vulns/vulns.go

```diff
@@ -369,6 +379,30 @@ func (affected *Affected) AttachExtractedVersionInfo(version cves.VersionInfo) {
 		affected.Ranges = append(affected.Ranges, gitRange)
 	}
 
+	for repo, commits := range repoToLimitCommits {
```

Comment:
Can we consolidate everything (all  fixed, limit, last_affected events) in the one events array?

Reply:
I think I've done this in e45b64c29992a1047f2ad866f94911a2c9e439fc, PTAL?